x-pack/plugins/infra/common/utils/index_to_alphabetical_label.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const idxToAlphabeticalLabel = (idx: number) => {
+  const labels = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
Wouldn't we need to consider i18n here, as different languages have different alphabets?

Reply:
We could label them numerically instead, here's the issues around that:

I think whatever we display as condition labels should be programmatically accessible in the advanced `context` variables. That's why I added the ability to access `context.value.conditionA, conditionB, conditionC` etc. This *might* work regardless of language if they're just symbols? Maybe there's an international keyboard that doesn't have a Q key or something, though.

You can still access them through `context.value.condition0, condition1, condition2...` but @hbharding and I thought it'd be awkward to start the labeling in the visual UI at `0`, and it'd be a huge backwards compatibility nightmare to start the `context` numbering at `1`.

So my opinion on labeling them numerically, starting at 0, is:

- **Against:** Numbering things starting at 0 in a UI is strange, and we'd only be doing it to accommodate a "power user" feature
- **For:** Our users are all VERY COMPUTERY and might not be put off by things starting at 0